base_classes/NXcoordinate_system.nxdl.xml

```diff
@@ -0,0 +1,208 @@
+
+          McStas is used if no instance of NXcoordinate_system is specified.
+          However, for the sake of clarity, even in this case it is better
+          to be explicit like for every other coordinate system definition
+          to support users with interpreting the content and logic behind
+          every instance of the tree.
+
+        * 1; if only one :ref:`NXcoordinate_system` is defined, it should be
+          placed as high up in the tree hierarchy (ideally right below an
+          instance of NXentry) of the application definition tree as possible.
+          This coordinate system shall be named such that it is clear how this
+          coordinate system is typically referred to in a community. For the
+          NeXus McStas coordinate system, it is advised to call it ``mcstas``
+          for the sake of improved clarity.
+
+          If this is the case, it is assumed that this ``NXcoordinate_system``
+          overwrites the NeXus default McStas coordinate system, i.e. users
+          can thereby conveniently and explicitly specify the
+          coordinate system that they wish to use.
+
+          This case has the advantage that it is explicit and offers no
+          ambiguities. However, in reality typically multiple coordinate
+          systems have to be mastered especially for complex multi-signal
+          modality experiments.
+
+        * 2 and more; as soon as more than one :ref:`NXcoordinate_system`
+          is specified somewhere in the tree, different interpretations are
+          possible as to which of these coordinate systems
+          apply or take preference.
+          While these ambiguities should be avoided if possible, the
+          opportunity for multiples instances enables to have coordinate
+          system conventions that are specific for some part of the NeXus
+          tree. This is especially useful for deep groups where
+          multiple scientific methods are combined or cases where having a
+          definition of global conversion tables how to convert between
+          representations in different coordinate systems is not desired
+          or available for now.
+
+          If this case is realized, the best practice is that in every
+          case where a coordinate system should be referred to the respective
+          group has a ``depends_on`` field, which resolves the possible
+          ambiguities which specific coordinate systems is referred to.
+
+          In the case of two or more instances of ``NXcoordinate_system`` it
+          is advised to specify the relationship between the two coordinate
+          systems by using the :ref:`NXtransformations` group within
+          ``NXcoordinate_system``.
+
+        In any case, users are encouraged to write explicit and clean
+        ``depends_on`` fields in all groups that encode information for which
+        the interpretation of coordinate systems is relevant. If these
```

Comment:
suggestion to add "and transformations" between "coordinate systems" and "is relevant" ?